src/static/js/modules/jquery/cf_formValidator.js

```diff
@@ -0,0 +1,185 @@
+/* ==========================================================================
+   jquery.cf_formValidator
+   ========================================================================== */
+
+'use strict';
+
+var $ = require( 'jquery' );
+var validate = require( 'validate' );
```

Comment:
Unless this is exported from the module, I would suggest prefixing it with an underscore to denote it's a private variable only used in this module.

Exceptions:
- the variable name is a known convention, like `$`.
- the variable references a module that only exports a constructor (in which case it'd be capitalized).